libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -417,6 +417,11 @@ namespace SohImGui {
                     CVar_SetS32("gMMBunnyHood", Game::Settings.enhancements.mm_bunny_hood);
                     needs_save = true;
                 }
+                
+                if (ImGui::Checkbox("Fix L&R Pause menu", &Game::Settings.enhancements.uniform_lr)) {
```

Comment:
Can you comment this out? I think we don't want to expose it on the UI and default it to true.

Reply:
This should be okay now I did changed something in GameSettings.cpp to be on by default 
because it seem changing the "if" in kaleido file to have a default value of 1 did not worked for me even after deleting .ini